packages/twenty-front/src/modules/command-menu/hooks/__tests__/useCommandMenu.test.tsx

```diff
@@ -143,4 +143,39 @@ describe('useCommandMenu', () => {
 
     expect(result.current.commandMenuCommands.length).toBe(2);
   });
+
+  it('should not add hotkeys to commands from custom objects', () => {
+    const { result } = renderHooks();
+
+    expect(result.current.commandMenuCommands).toHaveLength(1);
+
+    act(() => {
+      result.current.commandMenu.setObjectsInCommandMenu([
+        {
+          id: 'b88745ce-9021-4316-a018-8884e02d05ca',
+          nameSingular: 'task',
```

Comment:
Thanks for creating a test! Could you please pick another name for the custom object? Like Rocket or whatever you want. Because Task is a standard object so it can be misleading

Reply:
Changed